package.json

```diff
@@ -54,7 +54,7 @@
     "jsdoc-strip-async-await": "^0.1.0",
     "parallel-webpack": "^2.0.0",
     "uglifyjs-webpack-plugin": "^1.0.0-beta.1",
-    "webpack": "^3.0.0"
+    "webpack": "^3.5.4"
```

Comment:
Nuh! ^3.0.0, we only lock versions at release for the purpose of keeping them up2date to see if they break something. No reason to always update the numbers.